### PR TITLE
feat: Added new conditions and actions

### PR DESCRIPTION
**Pull request by bflorian**
Updated rule type definitions to reflect newly added features.

### CODE REVIEW

**Comment by vl4ds** [comment]
These features are currently in developer preview and as such have not been documented/made widely available. It's possible that we may make changes to them before adding them to api documentation (unlikely but possible) so wondering if it makes sense to have these exposed in this sdk? I am fine either way as long as this is considered and we are OK with that.

**Comment by vl4ds** [comment]
Apart from the changesTo, remainsEqual, wasEqual conditions it LGTM

File: src/endpoint/rules.ts [hunk 1]
```diff
@@ -48,7 +67,13 @@ export interface Condition {
 	lessThan?: SimpleCondition
 	lessThanOrEquals?: SimpleCondition
 	between?: BetweenCondition
-	changes?: SingleOperandCondition
+	changesTo?: SimpleCondition
+	dropsBelow?: SimpleCondition
```
[the remark this answers is not in the document]
Comment (bflorian) [reply src/endpoint/rules.ts:71]: What about "changes". Is that functional now, and planned to be deprecated? Or is it currently an error?

**Comment by vl4ds** [comment]
changes is not implemented and should not be available.

File: src/endpoint/rules.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (bflorian) [reply src/endpoint/rules.ts:70]: OK, removed it along with changesTo, remainsEqual, and wasEqual

**Pull request merged**